bundles/core/org.eclipse.smarthome.core.test/pom.xml

```diff
@@ -23,22 +23,45 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.eclipse.tycho</groupId>
+        <groupId>${tycho-groupid}</groupId>
+        <artifactId>target-platform-configuration</artifactId>
+        <configuration>
+          <environments combine.self="override"></environments>
+          <dependency-resolution>
+            <extraRequirements>
+              <requirement>
+                <type>eclipse-feature</type>
+                <id>org.eclipse.smarthome.feature.test</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>org.eclipse.equinox.ds</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>org.eclipse.equinox.event</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>com.eclipsesource.jaxrs.jersey-min</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>com.eclipsesource.jaxrs.jersey-all</id>
```

Comment:
I think you mixed my replies to "Why is it needed to have jersey here and in the other tests?" and "Same question for the test feature?".

If you know, that jersey is definitely not needed, I will remove it. Sure!

And why should we remove the dependency to the test feature?
What is the intention of this feature, if not "use this for tests"?